jekyll/_cci2/local-cli.md

```diff
@@ -22,26 +22,28 @@ You can also run `circleci` commands in your `config.yml` file for jobs that use
 
 ## Installing
```

Comment:
Remove this heading (better to just add the word to the following two headings instead of stacking the heads).

Reply:
I disagree based on how HTML documents are suppose to be structured but I made the change.